### PR TITLE
Add CommandRPC support

### PR DESCRIPTION
**Pull request by qiyuangong**
submitCommand for command submit.
	getCommandInfo for getting commandinfo with given commandID.
	listCommandinfs for getting all commands executing.
	activateCommand for activating(insert it to queue) an given command.
	deleteCommand for delete a command from queue, if this command is running, then kill it.

### CODE REVIEW

**Comment by qiyuangong** [comment]
@littlezhou Please check if these API satisfy our Demo.

File: hadoop-ssm-project/src/main/java/org/apache/hadoop/ssm/SSMRpcServer.java [hunk 1]
```diff
@@ -163,4 +164,34 @@ public void disableRule(long ruleID, boolean dropPendingCommands) throws IOExcep
     checkIfActive();
     ssm.getRuleManager().DisableRule(ruleID, dropPendingCommands);
   }
+
+  @Override
+  public long submitCommand(CommandInfo commandInfo) throws IOException {
+    checkIfActive();
```
Comment (littlezhou) [inline hadoop-ssm-project/src/main/java/org/apache/hadoop/ssm/SSMRpcServer.java]: remove

File: hadoop-ssm-project/src/main/java/org/apache/hadoop/ssm/SSMRpcServer.java [hunk 2]
```diff
@@ -163,4 +164,34 @@ public void disableRule(long ruleID, boolean dropPendingCommands) throws IOExcep
...
+  }
+
+  @Override
+  public List<CommandInfo> listCommandInfo() throws IOException {
+    checkIfActive();
+    return ssm.getCommandExecutor().listCommandsInfo();
+  }
+
+  @Override
+  public void activateCommand(long commandID) throws IOException {
```
Comment (littlezhou) [inline hadoop-ssm-project/src/main/java/org/apache/hadoop/ssm/SSMRpcServer.java]: any interface for disable ?

File: hadoop-ssm-project/src/main/java/org/apache/hadoop/ssm/SSMRpcServer.java [hunk 3]
```diff
@@ -163,4 +164,34 @@ public void disableRule(long ruleID, boolean dropPendingCommands) throws IOExcep
...
+  }
+
+  @Override
+  public CommandInfo getCommandInfo(long commandID) throws IOException {
+    checkIfActive();
+    return ssm.getCommandExecutor().getCommandInfo(commandID);
+  }
+
+  @Override
+  public List<CommandInfo> listCommandInfo() throws IOException {
```
Comment (littlezhou) [inline hadoop-ssm-project/src/main/java/org/apache/hadoop/ssm/SSMRpcServer.java]: better to have filter condition

**Pull request merged**